### ISSUE TITLE
Unsupervised speech representation learning using WaveNet autoencoders

### ISSUE THREAD

**Issue by npresseller** [npresseller/Capstone-Bibliography#8]
### Title
Unsupervised speech representation learning using WaveNet autoencoders

### URL
https://arxiv.org/abs/1901.08810

### Summary 
The article discusses unsupervised extraction of meaningful latent representations of speech using autoencoding neural networks to capture high level semantic content while being invariant to confounding low level details in the signal. Three variants of autoencoder models are compared and analyzed based on speaker independence, ability to predict phonetic content, and accuracy of individual spectrogram frame reconstruction. A regularization scheme is introduced to focus on the phonetic content of the utterance, and the performance is reported comparable to top entries in the ZeroSpeech 2017 task.

### Key Points 
Neural Nets-
Neural networks are hierarchical information processing
models that are typically implemented using layers of computational units. Each layer can be interpreted as a feature extractor
whose outputs are passed to upstream units. Especially in the visual domain, features learned with neural networks have been shown to create a hierarchy of visual atoms that match some properties of the visual cortex. Similarly, when applied to audio waveforms, neural networks have been shown to learn auditory-like frequency decompositions on music and speech in their lower layers.

Unsupervised learning using autoencoders for feature learning-
Autoencoders are neural networks that aim to reconstruct their inputs and use an encoding network to extract a latent representation, which is then passed through a decoding network to recover the original data. The latent representation should preserve the important features of the data while being easier to analyze and work with. This is achieved through the application of regularization techniques and constraints or bottlenecks. The autoencoder's representation is subject to two competing forces: providing the decoder with necessary information for perfect reconstruction and discarding some information to prevent trivial inversion.

Reducing the dimensionality of the latent representation can be a constraint applied to the latent vectors, making the autoencoder function as a nonlinear version of linear low-rank data projections, such as PCA or SVD. However, such representations may be difficult to interpret because the reconstruction of an input depends on all latent features. In contrast, dictionary learning techniques, such as sparse and non-negative decompositions, express each input pattern using a combination of a small number of selected features out of a larger pool, facilitating their interpretability. Discrete feature learning using vector quantization is an extreme form of sparseness, where the reconstruction uses only one element from the dictionary. Overall, this paper presents various approaches to unsupervised feature learning and discusses the advantages and drawbacks of each method.

Autoencoders for sequential data-
The paper discusses the use of generative models to exploit local dependencies in sequential data such as speech or text. Autoregressive models, which predict the next observation based on recent history, are successful in this regard. However, they do not explicitly produce latent representations of the data. To address this, the paper proposes combining an autoregressive sequence generation model with an encoder that extracts latent representations. The encoder can emit a single latent vector and feed it to an autoregressive decoder or periodically emit vectors of latent features to be consumed by the decoder.

The paper also highlights a challenge in training mixed latent variable and autoregressive models, which is the risk of latent space collapse. This occurs when the decoder learns to ignore the constrained latent representations and only uses the unconstrained signal coming through the autoregressive path. The paper proposes two methods to prevent this: annealing the weight of the KL term and using the free-information formulation in Eq. (2) for the VAE, and relying on the hyperparameter for the KL term in the VQ-VAE. The paper defers further discussion on this topic to a later section.

### Citation
@article{DBLP:journals/corr/abs-1901-08810,
  author    = {Jan Chorowski and
               Ron J. Weiss and
               Samy Bengio and
               A{\"{a}}ron van den Oord},
  title     = {Unsupervised speech representation learning using WaveNet autoencoders},
  journal   = {CoRR},
  volume    = {abs/1901.08810},
  year      = {2019},
  url       = {http://arxiv.org/abs/1901.08810},
  eprinttype = {arXiv},
  eprint    = {1901.08810},
  timestamp = {Sat, 02 Feb 2019 16:56:00 +0100},
  biburl    = {https://dblp.org/rec/journals/corr/abs-1901-08810.bib},
  bibsource = {dblp computer science bibliography, https://dblp.org}
}
